config/api.yml

```diff
@@ -1272,6 +1272,24 @@
       :delete:
       - :name: delete
         :identifier: chargeback_rates_delete
+  :regions:
+    :description: Regions
+    :identifier: region
+    :options:
+    - :collection
+    :verbs: *gp
```

Comment:
not today.  This post is just for the bulk query implemented.

But shortly, it will also be used for updating settings.